src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -223,7 +226,7 @@ private static BasicStudyInfos toBasicInfos(BasicStudyEntity entity) {
     public Mono<StudyEntity> createStudy(String studyName, Mono<FilePart> caseFile, String description, String userId, Boolean isPrivate) {
         return insertStudyCreationRequest(studyName, userId, isPrivate)
                 .then(importCase(caseFile).flatMap(uuid ->
-                     Mono.zip(persistentStore(uuid), getCaseFormat(uuid))
```

Comment:
Should we not treat the case of case imports with errors ?

Reply:
You're right. This case is now treated